lib/web_ui/test/text_editing_test.dart

```diff
@@ -676,6 +676,54 @@ void main() {
       expect(spy.messages, isEmpty);
     });
   });
+
+  group('SelectionChangeDetection', () {
+    SelectionChangeDetection _selectionChangeDetection;
+
+    setUp(() {
+      _selectionChangeDetection = SelectionChangeDetection();
+    });
+
+    test('Change detected on an input field', () {
+      final InputElement input = document.getElementsByTagName('input')[0];
+
+      input.value = 'foo\nbar';
+      input.setSelectionRange(1, 3);
+
+      expect(_selectionChangeDetection.hasChangeDetected(input), true);
```

Comment:
Should we add another call immediately after this, and expect it to return false? (same for all other tests).

Reply:
done.